tools/panic.py

```diff
@@ -0,0 +1,24 @@
+"Handle a critical error."
+
+import sys
+
+
+def panic_msg(exc: Exception) -> None:
```

Comment:
I decided to create a `tools.panic` module, to handle a "possible" critical error.